### ISSUE TITLE
"Multilist with Search" field type missing from standard configuration

### ISSUE THREAD

**Issue by mattbarry** [blipson89/Synthesis#100]
**Do you want to request a *feature* or report a *bug*?**

It's not a bug, it's a feature!

**What is the current behavior?**

Synthesis is mapping the "Multilist with Search" field type to TextField/ITextField with the OOTB config. I'm assuming this is the default when a field mapping can't be found.

**If the current behavior is a bug, please provide the steps to reproduce.**

Just add a "Multilist with Search" field to any template and generate the Syntheis model.

**What is the expected behavior?**

The "Multilist with Search" field type gets mapped to the ItemReferenceListField/IItemReferenceListField types. I patched it into my Synthesis config and confirmed it works as expected:
```
<fieldMappings hint="raw:AddMapping">
  <map field="Multilist with Search" type="Synthesis.FieldTypes.ItemReferenceListField, Synthesis" interface="Synthesis.FieldTypes.Interfaces.IItemReferenceListField, Synthesis" />
</fieldMappings>
```

**Please mention your Sitecore version and Synthesis version.**
Sitecore 9.3.0 and Synthesis 9.1.2.1

**Comment by blipson89** [comment]
By the way, congrats on logging the

# :100:<sup>th</sup> Issue

:tada:

**Comment by blipson89** [comment]
Released in [9.1.4](https://github.com/blipson89/Synthesis/releases/tag/9.1.4)
